test/components/slots.test.ts

```diff
@@ -13,7 +13,25 @@ beforeAll(() => {
 });
 
 test('should have correct region definitions', () => {
-  expect(component.properties).toEqual([]);
+  expect(component.properties).toEqual([
+    {
+      name: 'media',
+      description: 'Media content',
+      optional: true,
+      type: '{ content: React.ReactNode; }',
+      inlineType: {
+        name: '{ content: React.ReactNode; }',
+        properties: [
+          {
+            name: 'content',
+            optional: true,
```

Comment:
A keen reviewer might ask why there is `optional: true` when the `content` above does not have a `?`

This is happening because `undefined` is assignable to `ReactNode`, so `{media: undefined}` is supported by type so it is technically optional